packages/react-kitchen-sink/src/rxjs/__tests__/schemaDocumentSnapshotSubject.test.ts

```
@@ -0,0 +1,36 @@
+import { type DocumentSnapshot } from '@firebase/firestore'
```

ESLint

[error] Filename is not in kebab case. Rename it to `schema-document-snapshot-subject.test.ts`.

(unicorn/filename-case)